tests/types/otp-test.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "https://packageurl.org/schemas/purl-test.schema-0.1.json",
+  "tests": [
+    {
+      "description": "Parse test for <class 'type'> PURL",
+      "test_group": "base",
+      "test_type": "parse",
+      "input": "pkg:otp/asn1@5.4.1?platform=linux&arch=amd64&repository_url=https:%2F%2Fgithub.com%2Ferlang%2Fotp&vcs_url=git%20https:%2F%2Fgithub.com%2Ferlang%2Fotp.git#src/asn1ct.erl",
+      "expected_output": {
+        "type": "otp",
+        "namespace": null,
+        "name": "asn1",
+        "version": "5.4.1",
+        "qualifiers": {
+          "platform": "linux",
+          "arch": "amd64"
+        },
+        "subpath": "src/asn1ct.erl"
+      },
+      "expected_failure": false,
+      "expected_failure_reason": null
+    },
+    {
+      "description": "Rountrip test for <class 'type'> PURL",
```

Comment:
"Rountrip" mis-spelled in description - should be "Roundtrip"

Reply:
@maennchen My apologies for the nitpicking but Roundtrip is misspelled in 2 places in the test file. I suspect that you inherited the error from older versions of other test cases.

Reply:
@mjherzog Fixed